src/frontends/onnx/frontend/src/op/com.microsoft/range.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "core/operator_set.hpp"
+#include "openvino/frontend/exception.hpp"
+#include "openvino/op/ops.hpp"
+
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace com_microsoft {
+namespace opset_1 {
+ov::OutputVector range(const ov::frontend::onnx::Node& node) {
+    auto nodes = node.get_ov_inputs();
+    FRONT_END_GENERAL_CHECK(nodes.size() >= 2 && nodes.size() <= 3,
+                            "Range takes 2 or 3 inputs. Provided " + std::to_string(nodes.size()));
+    auto start = nodes.at(0);
```

Comment:
And use nodes[0] and nodes[1] for inputs which availability was checked before.

Reply:
done